scripts/get_deps

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+function die
```

Comment:
is this function used?

Reply:
Yes, immediately below this if bad parameters are passed.

Reply:
I think I meant to leave this comment in the `postinst` script.. the `die` function isn't used in that script, right?